app/controllers/stories_controller.rb

```diff
@@ -141,7 +141,8 @@ def handle_podcast_index
     @podcast_index = true
     @article_index = true
     @list_of = "podcast-episodes"
-    @podcast_episodes = @podcast.podcast_episodes.reachable.order("published_at DESC").limit(30)
+    @podcast_episodes = PodcastEpisodeDecorator.decorate_collection(@podcast.podcast_episodes.
```

Comment:
I believe you can do that with just adding `.decorate` to the query. Haven't noticed that on the first review.

Reply:
would you give me example. i'm new to ruby

Reply:
I mean `@podcast.podcast_episodes.reachable.order("published_at DESC").limit(30).decorate`

Reply:
Updated, thanks